src/pages/SingleCell/CustomToolTip.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const CustomToolTip: React.FC = () => {
+    return (
+        <div
+            style={{
+                backgroundColor: 'white',
+                padding: '10px',
+                border: '1px solid black',
+            }}
+        >
+            <p>Type of Cell: Suraj</p>
```

Comment:
Suraj?

Reply:
Actually, it was made for testing purposes and hasn't been used in any component. I have removed this component in subsequent commits.